src/pybind/mgr/prometheus/module.py

```diff
@@ -1294,14 +1297,13 @@ def get_rbd_stats(self) -> None:
                 namespace_name = s[1]
 
             if pool_name == "*":
-                # collect for all pools
-                osd_map = self.get('osd_map')
-                for pool in osd_map['pools']:
-                    if 'rbd' not in pool.get('application_metadata', {}):
-                        continue
-                    pool_keys.append((pool['pool_name'], namespace_name))
+                # collect for all existing pools
```

Comment:
"Existing" here is redundant -- just as it was in the variable name:
```suggestion
                # collect for all pools
```